lib/dark_worlds_server/runner_supervisor/runner.ex

```diff
@@ -482,4 +482,11 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
          {:loot, killed_id} | tail
        ]),
        do: [%{killed_by: 1111, killed: killed_id} | transform_killfeed_to_game_killfeed(tail)]
+
+  def transform_effects_to_game_effects([]), do: []
+
+  def transform_effects_to_game_effects([{%{name: "damage_outside_area"}, :zone} | tail]),
+    do: [{6, 0} | transform_effects_to_game_effects(tail)]
```

Comment:
Love the magic number value 😄

Reply:
This should be fixed after we implement the new state, and remove the transform functions